odoo/fields.py

```diff
@@ -2370,6 +2370,8 @@ def _get_attrs(self, model_class, name):
         attrs = super()._get_attrs(model_class, name)
         # arguments 'selection' and 'selection_add' are processed below
         attrs.pop('selection_add', None)
+        if not attrs.get('group_expand'):
+            attrs['group_expand'] = '_default_group_expand'
```

Comment:
That's weird, as `group_expand` was only meant for many2one fields (in this model anyway).

Reply:
There's one of the group_expand tests that specifically passes it to a Selection field, to explicitly give it declaration order, it's in `test_read_group/tests/test_group_expand/`